index.js

```diff
@@ -171,4 +175,7 @@ app.all('*', (req, res) => {
   });
 });
 
-app.listen(PORT, HOST, () => console.log(`Smart mockserver running at ${HOST}:${PORT}`));
+http.createServer(app).listen(PORT, HOST, () => console.log(`Smart mockserver running at ${HOST}:${PORT}`));
+https
+  .createServer({ key: readFileSync('cert/localhost.key'), cert: readFileSync('cert/localhost.crt') }, app)
+  .listen(1081, HOST, () => console.log(`Smart mockserver running at ${HOST}:${1081}`));
```

Comment:
First of all: Add an environment variable to define the HTTPS port. 
Second: That variable should default to `undefined` - check for the existence of given variable and only create an HTTPS server if given, because most of the time we don't need that feature anyways.

Reply:
Like that?